site/app/templates/forum/CreatePost.twig

```diff
@@ -119,6 +120,22 @@
                 (<i>Last edit at {{ edit_date }}</i>)
             {% endif %}
         </span>
+
+        <a class="pinned-thread key_to_click" tabindex="0" data-testid="thread-dropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" title="Options">
```

Comment:
`key_to_click` was deprecated in PR #10740 since button elements have the same functionality by default. `key_to_click` could be removed if the `<a>` elements are replaced by `<button>` elements.
This should also be a button since only actual links should use the `<a>` tag.

I'm not sure if this can be done without changing the style of the buttons, and a lot of Submitty uses `<a>` anyway, so feel free to ignore this comment.

Reply:
Ye I tried the button looks a bit odd